.github/workflows/test.yml

```diff
@@ -139,7 +144,7 @@ jobs:
           path: .jest-cache
           key: jest-integration-${{ env.GITHUB_SHA }}
           restore-keys: jest-integration-
-      - run: yarn cover:integration:${{ matrix.part }} --ci --cacheDirectory .jest-cache || yarn cover:integration:${{ matrix.part }} --ci --cacheDirectory .jest-cache -f
+      - run: yarn cover:integration:${{ matrix.part }} --ci --cacheDirectory .jest-cache --shard ${{ matrix.shards }}/8 || yarn cover:integration:${{ matrix.part }} --ci --cacheDirectory .jest-cache --shard ${{ matrix.shards }}/8 -f
```

Comment:
I think we don't need `matrix.part` in this case

Reply:
I think part is redundancy assurance no?

Reply:
We use this for emulate shard, this is of course far from it, just a simple solution to split tests

Reply:
Looks at package.json :smile:

Reply:
Okay will remove once I figure out how to use shard on the integration test commands.